fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/executable/StringArithmetic.java

```diff
@@ -208,63 +208,63 @@ public static Expression ltrimVarcharVarchar(StringLikeLiteral first, StringLike
      * Executable arithmetic functions rtrim
      */
     @ExecFunction(name = "rtrim")
-    public static Expression rtrimVarchar(StringLikeLiteral first) {
+    public static Expression rtrim(StringLikeLiteral first) {
         return castStringLikeLiteral(first, trimImpl(first.getValue(), " ", false, true));
     }
 
     /**
      * Executable arithmetic functions rtrim
      */
     @ExecFunction(name = "rtrim")
-    public static Expression rtrimVarcharVarchar(StringLikeLiteral first, StringLikeLiteral second) {
+    public static Expression rtrimVarchar(StringLikeLiteral first, StringLikeLiteral second) {
```

Comment:
why remove signature in name? and give it a meaningless name?

Reply:
because Signature in name is included by StringLikeLiteral, we need this signature before to indicate it is VarcharLiteral or StringLiteral. But now we can use StringlikeLiteral to include them all. Keep one in function to indicate the difference between  rtrim and rtrimVarchar